src/main/java/picard/sam/markduplicates/MarkDuplicatesForFlowHelper.java

```diff
@@ -59,6 +60,7 @@ public class MarkDuplicatesForFlowHelper implements MarkDuplicatesHelper {
     public static final char[]  CLIPPING_TAG_CONTAINS_AQ = {'A', 'Q'};
     public static final char[]  CLIPPING_TAG_CONTAINS_QZ = {'Q', 'Z'};
 
+    public static final int DIST_FROM_END = 10;
```

Comment:
It's worth mentioning that the distance from the end is hard coded as 10 bases for the `FLOW_END_QUALITY_STRATEGY`

Reply:
Mentioned in the parameter description. Is this OK?

Reply:
Yup, perfect